app/controllers/application_controller.rb

```diff
@@ -13,7 +13,9 @@ def is_admin
 	def set_locale
 	  I18n.locale = params[:locale]
 	  I18n.locale ||= request.headers['locale'] if request.headers['locale'].present?
-	  I18n.locale ||= :he
+    # Avriel edit: added locale by cookie
```

Comment:
in general, you don't need to add a comment that you did the change. for that we have git blame